.travis.yml

```diff
@@ -6,22 +6,43 @@ python:
 git:
     submodules: false
 
+env:
+    global:
+        - PYTHON_VERSION=3.6
+        - SETUP_CMD='test'
+        - MAIN_CMD='python setup.py'
+        - CONDA_DEPENDENCIES='Cython astropy scipy h5py matplotlib mpmath pytest'
+        - CONDA_ALL_DEPENDENCIES='Cython astropy scipy h5py matplotlib mpmath pytest'
+        - PIP_DEPENDENCIES='roman lmfit colorama codecov'
+
+matrix:
+
+    # Don't wait for allowed failures
+    fast_finish: true
+
+    include:
+
+        - env: PYTHON_VERSION=3.6 SETUP_CMD='test --coverage'
+
+        - env: PYTHON_VERSION=3.7 SETUP_CMD='test'
+
 # Install dependencies
 install:
         - git clone git://github.com/astropy/ci-helpers.git --depth 1
         - source ci-helpers/travis/setup_conda.sh
-        - pip install -r requirements/requirements.txt
-        - pip install codecov coveralls flake8 pytest-cov pytest-astropy
+#        - pip install -r requirements/requirements.txt
+#        - pip install codecov coveralls flake8 pytest-cov pytest-astropy
```

Comment:
Do we still need `coveralls` here since we switched primarily to `codecov`?  I'm not sure if `codecov` uses `coveralls` or something like that.

Reply:
Oh yeah that's a mistake right there. Thanks, I'll get rid of it!

I also saw a neat way to simplify our requirements files on sunpy so I guess I'll do that as well while I'm at it.